cmd/apiserver/app/server/options.go

```diff
@@ -41,6 +41,7 @@ type ServiceCatalogServerOptions struct {
 	EtcdOptions *EtcdOptions
 	// TPROptions are options for serving with TPR as the backing store
 	TPROptions *TPROptions
+	StopCh     <-chan struct{}
```

Comment:
@jpeeler this needs to get populated when a new options is created in `./cmd/apiserver/app/server/server.go`